filebeat/module/apache/error/ingest/pipeline.json

```
@@ -25,6 +25,24 @@
                 "ignore_failure": true
             }
         },
+        {
+            "date": {
+                "if": "ctx.event.timezone != null",
+                "field": "@timestamp",
```

Does it work correctly with timezone is not UTC? We may need to use the original timestamp and formats because the parsed timestamp already has the UTC timezone.
```suggestion
                "field": "apache.error.timestamp",
                "target_field": "@timestamp",
                "formats": [
                    "EEE MMM dd H:m:s yyyy",
                    "EEE MMM dd H:m:s.SSSSSS yyyy"
                ],    
```

This may be wrong in other modules too :slightly_frowning_face: See https://github.com/elastic/beats/pull/13308

---

@jsoriano Hmm how about in this json file line 18 right above this part? It converts `apache.error.timestamp` into `@timestamp`. Maybe I misunderstand it, I thought if `event.timezone` is not null, then both `date` pipelines will be ran?

---

Yes, if there is an `event.timezone`, date is parsed again with the correct timezone, but this may be wrong, at least for some cases, after what we are seeing in #13308.